docs/time/index.rst

```diff
@@ -54,6 +54,10 @@ to UTC except for `Time from Epoch Formats`_.
 
 .. EXAMPLE END
 
+.. note:: The ``format`` for a |Time| or |TimeDelta| object impacts only the *output
```

Comment:
I like the intent, but think it is a bit confusing here to say the format impacts the output representation, when in the paragraph above it is stated it is used to interpret the input. Since this is aimed at new users, maybe we can combine the two? (And avoid a distracting note in the process.) I.e., expand the paragraph above into two, one each for format and scale, like:
```
The ``format`` argument specifies how to interpret the input values (e.g., ISO,
JD, or Unix time). By default, the same format will be used to represent the
time for output. One can change this format later as needed, but because this
is just for representation, that does not affect the internal representation
(which is always by two 64-bit values, the ``jd1`` and ``jd2`` attributes), nor
any computations with the object.

The ``scale``  argument specifies the `time scale`_ for the values 
(e.g., UTC, TT, or UT1). The ``scale`` argument is optional and defaults
to UTC except for `Time from Epoch Formats`_. It is possible to change
it (e.g., from UTC to TDB), which will cause the internal values to be 
adjusted accordingly.
```

Reply:
@mhvk - your version is fine, the question is whether this point is important enough to deserve a note. I agree that the note might be distracting, but that is somewhat on purpose. The issue author in #15297 seemed to have a fundamental misunderstanding of the role of the format, but I don't know how common that is. I suspect that most users would gloss over text that occurs at the end paragraph in your version, so even if the docs are formally complete they might not be effective.

Either way is fine by me.

Reply:
I agree with your worry, but think it is quite likely people just do not read the documentation anyway. So, I say we aim for those who do read it -- let's go with my version!

But to help less patient people, should we adjust the `Time` docstring? Right now, it just says
```
    format : str, optional
        Format of input value(s)
```

We could change it to
```
    format : str, optional
        Format of input value(s), specifying how to interpret them (e.g., ISO,
        JD, or Unix time). By default, the same format will be used for output
        representation.
```

Reply:
Done!